src/core/results_interface.jl

```diff
@@ -0,0 +1,20 @@
+# Declaring functions for IS.Results interface
+function get_variables(result::T) where {T <: IS.Results}
+    return IS.get_variables(result)
+end
+
+function get_total_cost(result::T) where {T <: IS.Results}
+    return IS.get_total_cost(result)
+end
+
+function get_optimizer_log(result::T) where {T <: IS.Results}
+    return IS.get_optimizer_log(result)
+end
+
+function get_time_stamp(result::T) where {T <: IS.Results}
+    return IS.get_time_stamp(result)
+end
+
+function write_results(results::T, save_path::String; kwargs...) where {T <: IS.Results}
+    return IS.write_results(results, save_path; kwargs...)
+end
```

Comment:
Since these are identical signatures as what's in IS, you shouldn't need them.  Instead, you should be able to use this pattern:

```
import InfrastructureSystems: get_variables, get_total_cost, get_optimizer_log, get_time_stamp, write_results
export get_variables
export get_total_cost
export get_optimizer_log
export get_time_stamp
export write_results
```